byzcoin/heartbeat_test.go

```diff
@@ -28,7 +28,7 @@ func TestHeartbeat_Start(t *testing.T) {
 }
 
 func TestHeartbeat_Timeout(t *testing.T) {
-	t.Skip("https://github.com/dedis/cothority/issues/1473")
+	t.Skip("https://go.dedis.ch/cothority/v3/issues/1473")
```

Comment:
this should stay on github

Reply:
I guess we should have a redirection anyway

Reply:
The go,dedis.ch domain is just for the meta data that the go build tool uses. More redirects would just be more confusing.